content/en/developers/ide_plugins/vscode/_index.md

```diff
@@ -27,172 +29,164 @@ further_reading:
-For example, all [View in VS Code](#view-in-vs-code) links on the Datadog platform encode runtime version info, and the extension uses that to compute the corresponding line of code in your editor, taking into account version changes.
+## Static Code Analysis
 
-You can tweak the Code Delta settings to change how the matching algorithm works. In particular, you can modify the `Minimum Affinity` value, which determines the degree of confidence required to match lines.
+[Static Code Analysis][8] analyzes your code (locally) against predefined rules to detect and fix problems.
 
-## Static Analysis
+The extension runs Static Code Analysis rules on the source files you have open in your workspace. This allows you to detect and fix problems such as maintainability issues, bugs, or security vulnerabilities in the code before you commit your changes.
 
-The [Static Analysis][14] integration analyzes your code (locally) against predefined rules to detect and fix problems.
+Static Code Analysis supports scanning for many programming languages. For a complete list, see [Static Code Analysis Rules][9]. For file types belonging to supported languages, issues are shown in the source code editor, and you can directly apply suggested fixes.
 
-The Datadog extension runs [Static Analysis][14] rules on the source files you have open in your Workspace. The goal is to detect and fix problems such as maintainability issues, bugs, or security vulnerabilities in your code before you commit your changes.
+{{< img src="/developers/ide_plugins/vscode/static_analysis.mp4" alt="Preview of Static Analysis" style="width:100%" video=true >}}
 
-[Static Analysis][14] supports scanning for many programming languages. For a complete list, see [Static Analysis Rules][15]. For file types belonging to supported languages, issues are shown in the source code editor with the VS Code inspection system, and suggested fixes can be applied directly.
+### Get started with Static Code Analysis
 
-{{< img src="/developers/ide_plugins/vscode/static_analysis.mp4" alt="Preview of Static Analysis" style="width:100%" video=true >}}
+When you start editing a source file, the extension checks for [`static-analysis.datadog.yml`][10] at your source repository's root. It prompts you to create it if necessary.
 
-### Getting started
+{{< img src="/developers/ide_plugins/vscode/static-analysis-onboard.png" alt="Onboarding banner for setting up Static Code Analysis with Python files" style="width:75%;" >}}
 
-When you start editing a source file, the extension checks for [static-analysis.datadog.yml][22] at your source repository's root. It prompts you to create it if necessary.
+After you create the configuration file, the analyzer runs automatically in the background whenever you open a file. If you need to enable Static Code Analysis for a particular language, use the `Datadog: Configure Static Analysis Languages` command from the command palette (`Shift` + `Cmd` + `P`, then search for the command).
```

Comment:
Might help to have the Windows equivalent shortcut too. (Ctrl + Shift + P.)

Reply:
Added in 8f8ead49ec77974d01cb0a66a2865d98cbe0397d